lib/rpm_package.py

```diff
@@ -151,7 +151,7 @@ def _load(self):
 
             # load distro files
             files = self.package_data.get('files', {}).get(
-                distro_attrib_name, {}).get(self.distro.version, {})
+                distro_attrib_name, {}).get(self.distro.version, {}) or {}
```

Comment:
I don't get where this is needed. In cases where the `files` attribute of the yaml file is empty? Does this happen for any package?